api/app/Console/Commands/SendNotificationsApplicationDeadlineApproaching.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Pool;
+use App\Notifications\ApplicationDeadlineApproaching;
+use Illuminate\Console\Command;
+use Illuminate\Support\Carbon;
+use Throwable;
+
+class SendNotificationsApplicationDeadlineApproaching extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send-notifications:application-deadline-approaching';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send notifications to users with a draft application in a pool that is closing in three days from the current day.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->info('SendNotificationsApplicationDeadlineApproaching running at '.Carbon::now()->toDateTimeString());
+
+        $successCount = 0;
+        $failureCount = 0;
+
+        // server and database are running in UTC
+        $closingDayInPacific = Carbon::now('America/Vancouver')->addDays(3);
```

Comment:
I feel like this 3 days needs to be dynamic. 
Although we decided on 3 days before closing date for now , this may change. 
It has to be a message variable in the template as well. 

You started an application to [((opportunity title))](https://notification.canada.ca/services/57b8682c-8400-40bd-a6bc-453b0c0fdd21/templates/mark%20class='placeholder'%3E((job%20advertisement%20link))%3C/mark) which closes in 3 days, at midnight on ((closing date)).

Reply:
I considered this then figured it was probably over-engineering.  Let's bring it up in standup today to see if there's value there.